### PR TITLE
Sqs/sqcb changes

### PR DESCRIPTION
**Pull request by ganncamp**

### CODE REVIEW

File: softvis3d.properties [hunk 1]
```diff
@@ -9,8 +9,9 @@ defaults.mavenGroupId=de.rinderle.softvis3d
 defaults.mavenArtifactId=softvis3d
 
 1.3.0.description=Compatibility with SQ 10.X.
-1.3.0.sqVersions=[10.0,LATEST]
-1.3.0.date=2023-09-18
+1.3.0.sqs=[10.8,LATEST]
```
Comment (lukasz-jarocki-sonarsource) [inline softvis3d.properties:12]: date is missing here

File: softvis3d.properties [hunk 2]
```diff
@@ -9,8 +9,9 @@ defaults.mavenGroupId=de.rinderle.softvis3d
 defaults.mavenArtifactId=softvis3d
 
 1.3.0.description=Compatibility with SQ 10.X.
-1.3.0.sqVersions=[10.0,LATEST]
-1.3.0.date=2023-09-18
```
Comment (vilchik-elena) [inline softvis3d.properties:13]: extra line is removed, build is failing because of that

File: README.md [hunk 3]
```diff
@@ -51,14 +52,15 @@ Add file name (without `.properties` extension) to `plugins` value in https://gi
...
+* `[versionId].sqcb` =[compatibility with SonarQube Community Build. See 'Filling in sqVersions compatibility ranges' below]
 * `[versionId].date` =[release date with format: YYYY-MM-DD]
 * `[versionId].changelogUrl` =
 * `[versionId].downloadUrl` =
 
-Add `[versionId]` to the `publicVersions` list. Move to `archivedVersions` any versions with identical compatibility. See also 'Filling in sqVersions, publicVersions, and archivedVersions' below
+Add `[versionId]` to the `publicVersions` list. Move to `archivedVersions` any versions with identical compatibility. See also 'Filling in sqs, sqcb, publicVersions, and archivedVersions' below
 
-### Filling in `sqVersions` , `publicVersions` and `archivedVersions`
+### Filling in `sqs`, `sqcb` , `publicVersions` and `archivedVersions`
```
Comment (vilchik-elena) [inline README.md:63]:
```suggestion
### Filling in versions compatibility ranges (`sqs`, `sqcb` , `publicVersions` and `archivedVersions`)
```

File: README.md [hunk 4]
```diff
@@ -35,7 +35,8 @@ Provide the following meta values:
 For the initially listed version create the following block:
 
 * `[versionId].description` =[free text. Spaces allowed. No quoting required]
-* `[versionId].sqVersions` =[compatibility information. See 'Filling in sqVersions compatibility ranges' below]
+* `[versionId].sqs` =[compatibility with SonarQube Server. See 'Filling in sqVersions compatibility ranges' below]
+* `[versionId].sqcb` =[compatibility with SonarQube Community Build. See 'Filling in sqVersions compatibility ranges' below]
```
Comment (vilchik-elena) [inline README.md:39]:
```suggestion
* `[versionId].sqs` =[compatibility with SonarQube Server. See 'Filling in versions compatibility ranges' below]
* `[versionId].sqcb` =[compatibility with SonarQube Community Build. See 'Filling in versions compatibility ranges' below]
```

File: README.md [hunk 5]
```diff
@@ -51,14 +52,15 @@ Add file name (without `.properties` extension) to `plugins` value in https://gi
 Create a new block in the file with this format:
 
 * `[versionId].description` =[free text. Spaces allowed. No quoting required]
-* `[versionId].sqVersions` =[compatibility information. See 'Filling in sqVersions compatibility ranges' below]
+* `[versionId].sqs` =[compatibility with SonarQube Server. See 'Filling in sqVersions compatibility ranges' below]
+* `[versionId].sqcb` =[compatibility with SonarQube Community Build. See 'Filling in sqVersions compatibility ranges' below]
```
Comment (vilchik-elena) [inline README.md:56]:
```suggestion
* `[versionId].sqs` =[compatibility with SonarQube Server. See 'Filling in versions compatibility ranges' below]
* `[versionId].sqcb` =[compatibility with SonarQube Community Build. See 'Filling in versions compatibility ranges' below]
```

**Review by vilchik-elena — DISMISSED** [review]

File: softvis3d.properties (hunk 1 above)
Comment (ganncamp) [reply softvis3d.properties:12]: Actually, it's pushed onto the next line. But thanks for the review

**Comment by lukasz-jarocki-sonarsource** [comment]
We decided to split this PR into two parts. The first one about the SQCB is here:  https://github.com/SonarSource/sonar-update-center-properties/pull/635
Once it is merged I will rebase this PR.

**Comment by lukasz-jarocki-sonarsource** [comment]
I am rebasing/updating/resolving conflicts in this PR now.

**Review by christophelevis — APPROVED** [review]

**Pull request merged**